frontend/src/components/MeetingRoom/Toolbar/Toolbar.spec.tsx

```diff
@@ -15,24 +19,40 @@ vi.mock('react-router-dom', () => ({
 }));
 
 vi.mock('../../../hooks/useSpeakingDetector');
-vi.mock('../../../hooks/useIsSmallViewport');
-vi.mock('../../../utils/displayOnDesktop');
+vi.mock('../../../utils/isReportIssueEnabled');
+vi.mock('../../../hooks/useToolbarButtons');
 
 const mockUseSpeakingDetector = useSpeakingDetector as Mock<[], boolean>;
-const mockUseIsSmallViewport = useIsSmallViewport as Mock<[], boolean>;
-const mockDisplayOnDesktop = displayOnDesktop as Mock<[], '' | 'md:inline'>;
+const mockIsReportIssueEnabled = isReportIssueEnabled as Mock<[], boolean>;
+const mockUseToolbarButtons = useToolbarButtons as Mock<
+  [UseToolbarButtonsProps],
+  UseToolbarButtons
+>;
 
 describe('Toolbar', () => {
   beforeEach(() => {
     (useLocation as Mock).mockReturnValue({
       state: mockedRoomName,
     });
     mockUseSpeakingDetector.mockReturnValue(false);
+    mockIsReportIssueEnabled.mockReturnValue(false);
+    mockUseToolbarButtons.mockImplementation(
+      ({ numberOfToolbarButtons }: UseToolbarButtonsProps) => {
+        const renderedToolbarButtons: UseToolbarButtons = {
+          displayTimeRoomName: true,
```

Comment:
Nit: I like the literalness but maybe another moniker is possible.  Not idea what word can capture this too 🤷‍♂️ 

Naively, was thinking "Call details" or "Footer".  In any case, please feel free to disregard.

Reply:
I think it's okay for this one. The component displayed by this boolean is `TimeRoomName`.